apis/Google.LongRunning/Google.LongRunning/Operation.cs

```diff
@@ -28,8 +28,11 @@ namespace Google.LongRunning
     /// For simplicity, no methods on this type modify the proto message. Instead, to get up-to-date
     /// information you can use Refresh to obtain a new instance.
     /// </remarks>
-    /// <typeparam name="T">The response message type.</typeparam>
-    public class Operation<T> where T : IMessage<T>, new()
+    /// <typeparam name="TResponse">The response message type.</typeparam>
+    /// <typeparam name="TMetadata">The metata message type.</typeparam>
+    public sealed class Operation<TResponse, TMetadata>
+        where TResponse : IMessage<TResponse>, new()
+        where TMetadata : class, IMessage<TMetadata>, new()
```

Comment:
So that we can return null from the Metadata property if there isn't any metadata.